### PR TITLE
[ETCM-487] Sync info improvements.

### PR DESCRIPTION
**Pull request by conqeror**
# Description

- Display absolute numbers for state sync instead of percentage
- Add description to the popover for state sync
- Increase sync state refresh rate

### CODE REVIEW

File: src/common/SyncStatus.tsx [hunk 1]
```diff
@@ -14,19 +14,29 @@ interface SyncStatusProps {
   syncStatus: SynchronizationStatus
 }
 
+const getSyncBlocksProgress = (syncStatus: SynchronizationStatusOnline): number => {
+  return (syncStatus.currentBlock / syncStatus.highestKnownBlock) * 100
```
Comment (qwhex) [inline src/common/SyncStatus.tsx:19]: `highestKnownBlock` is always greater than 0?

File: src/common/wallet-state.ts [hunk 2]
```diff
@@ -658,7 +650,7 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
   useRecurringTimeout(async () => {
     await refreshSyncStatus()
     rendererLog.debug(`sync status`, option.getOrElseW(() => null)(syncStatusOption))
-  }, 3000)
+  }, 500)
```
Comment (qwhex) [inline src/common/wallet-state.ts]: Would it make sense to extract this and similar "config" constants into a separate file?
Online check interval is another good candidate.

File: src/translations/en/renderer.json [hunk 3]
```diff
@@ -156,8 +156,9 @@
     "syncStatus": {
       "syncConnecting": "Connecting",
       "fullySynced": "Fully Synced",
-      "syncingBlocks": "Syncing Blocks {{percentage}}%",
-      "syncingState": "Syncing State {{percentage}}%",
+      "syncingBlocks": "Syncing Blocks: {{percentage}}%",
+      "syncingState": "Syncing State: {{processed}} of {{total}}",
+      "syncingStateDescription": "Mantis Wallet is fetching all accounts state, including balances and smart contract codes",
```
Comment (qwhex) [inline src/translations/en/renderer.json:161]: What does "code" mean here? Status code?

Comment (conqeror) [reply src/translations/en/renderer.json:161]: It refers to the code which is executed by the smart contract.

File: src/common/SyncStatus.tsx (hunk 1 above)
Comment (conqeror) [reply src/common/SyncStatus.tsx:19]: Will fix :)

File: src/common/wallet-state.ts (hunk 2 above)
Comment (conqeror) [reply src/common/wallet-state.ts]: @qwhex For now, I put those into constants in the same file, which is the first step. Do you think we want to have those even user-configurable (by user config)? Or just separate them to some `constants.ts` file?

File: src/translations/en/renderer.json (hunk 3 above)
Comment (qwhex) [reply src/translations/en/renderer.json:161]: Not sure what would be a better phrasing in this case, let's just roll with it.

**Review by qwhex — APPROVED** [review]

**Pull request merged**